rct229/rulesets/ashrae9012019/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/get_min_oa_cfm_sch_zone.py

```diff
@@ -19,41 +17,49 @@ def get_min_oa_cfm_sch_zone(rmi, zone_id):
     list: An aggregated OA CFM hourly schedule for the zone (for each hour of the year, for each terminal unit, Terminal.minimum_outdoor_airflow is multiplied by Terminal.minimum_outdoor_airflow_multiplier_schedule, this product is summed across the terminal units for each hour of the year) .
     """
```

Comment:
I think you will need to initialize the `min_OA_CFM_schedule_for_zone` here, instead of inside the loop because you are returning the `min_OA_CFM_schedule_for_zone` at the end of the function.
The current implementation will run into error when the zone has no terminals, or none of the terminal have `minimum_outdoor_airflow_multiplier_schedule`.

Here are two options:
1. Pass in `is_leap_year` to the function - use `is_leap_year` to initialize the `min_OA_CFM_schedule_for_zone` with correct number of hours.
2. Set `min_OA_CFM_schedule_for_zone` to None as default. Also, add this behavior in the function comment "Returns None if the zone has no terminal or every terminal in the zone do not use minimum_outdoor_airflow_multiplier_schedule`

Could you check the rule definitions and let me know if returning None make sense?

Reply:
I'm not sure if I understood the no. 2. Nevertheless, I addressed this comment based on my understanding. Could you take a look at it?